sdk/healthinsights/Azure.Health.Insights.ClinicalMatching/samples/README.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
.NET SDK package organizes samples readme and code in an predefined structure. Please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/Create_Samples_Guidance.md and update the samples accordingly. Thanks